src/main/java/com/dungeoncrawler/Javiarenka/character/Character.java

```diff
@@ -1,17 +1,24 @@
 package com.dungeoncrawler.Javiarenka.character;
 
-public class Character {
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Character {
 
     private String name;
     private int hp;
-    private boolean isAlive = true;
+    private boolean isAlive;
+    private List<CharacterStatus> currentStatus;
+
 
     public Character() {
     }
 
     public Character(String name, int hp) {
         this.name = name;
         this.hp = hp;
+        this.isAlive = true;
+        this.currentStatus = new ArrayList<>();
```

Comment:
Elegancko. Jeszcze tylko zmieniłbym nazwę, bo teraz jak korzystamy z tej listy gdzieś spoza naszej klasy, to jej nazwa nie sugeruje, że to kolekcja. Lepiej będzie w liczbie mnogiej, na przyklad "allCharacterStatuses"

Reply:
pozmieniałam nazwy w polach i metodach, oprócz tego poszły jeszcze kosmetyczne zmiany => wszystko pójdzie w nowym pull-u